src/porepy/applications/md_grids/mdg_library.py

```diff
@@ -21,8 +21,8 @@ def square_with_orthogonal_fractures(
     grid_type: Literal["simplex", "cartesian", "tensor_grid"],
     meshing_args: dict[str, float],
     fracture_indices: list[int],
-    fracture_endpoints: Optional[list[[pp.number, pp.number]]] = None,
-    size: Optional[pp.number] = 1,
```

Comment:
I thought optional parameters should be explicitly marked as such. Is that wrong?

Reply:
`Optional` means the value can be `None.` In this case, the default will be `1`; you would have to explicitly write `(..., size=None, ...)` to get the effect of the `Optional`, and this is not really usage we want to account for.